hooks/post_gen_project.py

```diff
@@ -104,27 +112,31 @@ def remove_terraform_files(directory):
         }
 
         directory = "terraform/"
-        token_q = "Enter the personal access token for creating the organizational repo: "
-        os.environ['GITHUB_ORGANIZATION'] = "{{ cookiecutter.github_username }}"
-        os.environ['GITHUB_TOKEN'] = raw_input(token_q).lower()
+        token_q = ("Enter the personal access token for creating the organizational repo: ")
+        os.environ['GITHUB_ORGANIZATION'] = (
+            "{{ cookiecutter.github_username }}")
+        os.environ['GITHUB_TOKEN'] = input(token_q).lower()
 
-        with open( directory  + "/terraform.tfvars", "w") as f:
+        with open(directory + "/terraform.tfvars", "w") as f:
             f.write("name = " + "\"{{ cookiecutter.module_name }}\"" + "\n")
-            f.write("description = " + "\"{{ cookiecutter.short_description }}\"" + "\n")
+            f.write("description = "
```

Comment:
forget all of this linting stuff here. this is all string parsing, all going away in the conversion to json tfvars. focus on creating a single dictionary with all the variables as keys, then just dump the entire dict to json and write the file all at once.